apps/api/Dockerfile

```
@@ -1,34 +1,55 @@
+# Install pnpm
+RUN corepack enable && corepack prepare pnpm@10.8.1 --activate
+
+# Bring in Turborepo remote cache configuration
+ARG TURBO_API
+ENV TURBO_API=$TURBO_API
+
+ARG TURBO_TEAM
+ENV TURBO_TEAM=$TURBO_TEAM
+
+# The token comes from the GitHub secrets, not args
+RUN --mount=type=secret,id=TURBO_TOKEN
```

Hadolint (2.12.0)

[error] missing whitespace

(DL1000)